package.json

```diff
@@ -149,6 +150,7 @@
     "@commitlint/cli": "8.0.0",
     "@commitlint/config-conventional": "8.0.0",
     "@ng-select/ng-select": "3.7.2",
+    "@open-wc/karma-esm": "2.13.23",
```

Comment:
This package is a web component tool that adds native es module to karma.

Reply:
🏆